tool-plugins/intellij/getting-started/plugin-features/README.md

```diff
@@ -74,17 +62,29 @@ You can reformat the Ballerina codes by pressing `Ctrl`+`Alt`+`L`.
 
 ![alt text](images/Formatting.gif)
 
-## Path/Query Parameter validation
+## Parameter Information
+
+You can view the required parameters of a functions, actions, etc by pressing `Ctrl`+`P`.
+
+![alt text](images/ParameterInfo.gif)
+
+## Quick Documentation
+
+You can view the documentation of a functions, actions, etc by pressing `Ctrl`+`Q` or hovering over the element while pressing `Ctrl`. 
+
+![alt text](images/QuickDocumentation.gif)
+
+## Object/Record fields suggestions
 
-Path/Query parameter validation is available to improve usability.
+Fields are suggested inside the objects/records after initializing braces and the dot operator.
 
-![alt text](images/PathParameter.gif)
+![alt text](images/StructFields.gif)
 
-## Package Inspection
+## Annotation fields suggestions
 
-Package inspection is available to improve usability. This will suggest to add the package name if no package name is present. If an incorrect package name is present, it will suggest to change the package name. You can view these suggestions using `Alt`+`Enter`.
+Annotation field names will be suggested inside annotation attachments.
```

Comment:
Let's say "Annotation field names are suggested...". 

Avoid future tense whenever possible. Usually if you see "will" in a sentence it is an indication that future tense is used. At times we cannot avoid this, but apply in all other places if possible.

Reply:
Modified. Thanks for the explanation.